<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GoogleMapView</class>
 <widget class="QWidget" name="GoogleMapView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>662</height>
   </rect>
  </property>
  <property name="toolTip">
   <string></string>
  </property>
  <property name="windowTitle">
   <string>Google Map View</string>
  </property>
  <widget class="QLabel" name="Panel1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1030</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string></string>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <enum>Qt::AlignCenter</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="toolTip">
    <string></string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="BtnHome">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>0</y>
      <width>23</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string></string>
    </property>
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string></string>
    </property>
   </widget>
   <widget class="QLabel" name="Panel5">
    <property name="geometry">
     <rect>
      <x>978</x>
      <y>1</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Panel5</string>
    </property>
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <enum>Qt::AlignCenter</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="toolTip">
     <string></string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <widget class="QPushButton" name="BtnClose">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string></string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="Panel2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>22</y>
     <width>1030</width>
     <height>640</height>
    </rect>
   </property>
   <property name="text">
    <string>Panel2</string>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <enum>Qt::AlignCenter</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="toolTip">
    <string></string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="WebBrowser">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>1</y>
      <width>1028</width>
      <height>638</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
   </widget>
  </widget>
 </widget>
</ui>
